packages/client/components/GroupingKanban.tsx

```diff
@@ -92,14 +92,14 @@ const GroupingKanban = (props: Props) => {
     setActiveIdx(nextIdx)
   }, Times.REFLECTION_COLUMN_SWIPE_THRESH)
 
-  const openSpotlight = (reflectionId: string, reflectionRef: RefObject<HTMLDivElement>) => {
-    spotlightReflectionRef.current = reflectionRef.current
+  const openSpotlight = (reflectionGroupId: string, reflectionRef: RefObject<HTMLDivElement>) => {
+    spotlightGroupRef.current = reflectionRef.current
     openPortal()
     commitLocalUpdate(atmosphere, (store) => {
       const meeting = store.get(meetingId)
-      const reflection = store.get(reflectionId)
-      if (!reflection || !meeting) return
-      meeting.setLinkedRecord(reflection, 'spotlightReflection')
+      const reflectionGroup = store.get(reflectionGroupId)
+      if (!meeting || !reflectionGroup) return
```

Comment:
is this a normal case? If not it'd be nice to add a warning here so if that happens we can track what was wrong 💡

Reply:
these things should always exist, but using relay's imperative API it can't guarantee `store.get` is going to return something.

You could write something like this

```ts
      const meeting = store.get<GroupingKanban_meeting>(meetingId)!
```

because the meeting object is at least guaranteed to have the fields that this component has requested

Reply:
Personally, I feel more secure using the return statement instead of the bang, even if we know it should exist. The return statement adds an extra line & it could be confusing to devs wondering why it's there (although the bang could also be confusing), but if there's a strange bug in Relay, it could prevent a crash. 

I've pushed an update adding the meeting type. Happy to add the bang instead if anyone feels tension!